config/rubocop.yml

```diff
@@ -118,6 +126,9 @@ Style/Send:
 Style/StringMethods:
   Enabled: true
 
+Rails/RedundantReceiverInWithOptions:
+  Enabled: false
```

Comment:
I like the variant with explicit receiver more:
```ruby
validates :name, presence: true
with_options uniqueness: true do |unique|
  unique.validates :email, presence: true
end
```
than
```ruby
```ruby
validates :name, presence: true
with_options uniqueness: true do
  validates :email, presence: true
end
```
In the latter variant it's easier to miss the additional options.
WDYT?

Reply:
I think both options are fine style wise... so no need for a cop

Reply:
Hm... I can see the appeal of both of them. 😉 I don't mind disabling this cop, but then, Rubocop is about consistency, right? 🤔

Reply:
Sure, the second is shorter. But when the block is long enough, it's easier to not notice the extra options.

Reply:
Sure, in case of doubt, implicit is better then explicit.

Reply:
Uh... the other way around, of course.